components/org.wso2.carbon.identity.oauth.endpoint/pom.xml

```diff
@@ -323,7 +318,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.48</minimum>
```

Comment:
Any reason so reduce the test coverage? Can't we exclude the factory classes from test coverage?

Reply:
https://github.com/wso2/product-is/issues/22208